Automattic/wp-calypso
# Signup: Added suggestion field

A continuation of #19693. This PR adds a new vertical type question asking people what their site is about. The field is equipped with an list of suggestions which should help people complete their entries quicker. The results are recorded with tracks and saved into the state so that the field is populated should you want to return back to the step after moving ahead. 

![suggestions](https://user-images.githubusercontent.com/6981253/33218157-6058bb18-d109-11e7-8f8c-9900b91b042d.gif)

## Test
- Visit `/start/segment`
- Type `Designer` into the `what will your site be about?` field.
- Select `UI designer` from the suggestions > your field should be populated with `UI designer`. 
- Press continue to move ahead to the domain step. 
- Click `Back` on the domain step to ensure `UI designer` is still in the field when you get back to the new screen.

@markryall @taggon can you take a look?
